scipy/stats/_distn_infrastructure.py

```diff
@@ -199,7 +199,8 @@
 
 And compare the histogram:
 
->>> ax.hist(r, density=True, histtype='stepfilled', alpha=0.2)
+>>> ax.hist(r, density=True, bins='auto', histtype='stepfilled', alpha=0.2)
+>>> ax.set_xlim([x[0], x[-1]])
```

Comment:
```suggestion
>>> # ax.set_xlim([x[0], x[-1]])
```
Let's see which line does it.